run_anomaly/run_anomaly.py

```diff
@@ -61,7 +65,7 @@ def runAnomaly(options):
     reader = csv.reader(fin)
     csvWriter = csv.writer(open(options.outputFile,"wb"))
     csvWriter.writerow(["timestamp", "value",
-                        "raw anomaly score", "likelihood", "anomaly_score"])
+                        "_raw_score", "likelihood_score", "log_likelihood_score"])
```

Comment:
Note that I updated the names of these fields to be more clear and work directly with our plotting functions.